app/views/art_items/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= @art_item.name %></h1>
+
+<p>
+   <strong>Price :</strong> <%= @art_item.price %><br />
+</p>
+
+<hr />
+
+<p> <%= link_to 'Delete', @art_item, method: :delete %></p>
```

Comment:
Since in art_items_controller.rb the defined method is "destroy" shouldn't it be destroy instead of "delete" here too?

Reply:
The router uses the HTTP verb and URL to know which action in the controller you want to use, in this case delete invokes destroy.